Source/WTF/wtf/ObjectIdentifier.cpp

```diff
@@ -31,7 +31,7 @@ namespace WTF {
 
 uint64_t ObjectIdentifierBase::generateIdentifierInternal()
 {
-    static uint64_t current;
+    static uint64_t current = 0;
```

Comment:
This is a really peculiar warning. Why does `cppcheck` care about this?

Reply:
All it said is:
```
Source/WTF/wtf/ObjectIdentifier.cpp:34:21: style: Variable 'current' is not assigned a value. [unassignedVariable]
    static uint64_t current;
```

I would have expected it to get zero-initialized. It is possible this is a false positive.